modules/about-oadp-update-channels.adoc

```diff
@@ -9,23 +9,25 @@
 
 When you install an OADP Operator, you choose an _update channel_. This channel determines which upgrades to the OADP Operator and to Velero you receive. You can switch channels at any time.
 
-There are three update channels:
+The available update channels are:
```

Comment:
This needs to be  slightly rewritten - see [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) on lead-in wording for lists (to avoid translation problems). (So for example, "The following update channels are available:" or similar.)